### ISSUE TITLE
Preventing nested content in self-closing elements

### ISSUE THREAD

**Issue by pixelastic** [pugjs/pug-lint#134]
The following (invalid) code will pass through `pug-lint`:

```pug
body
  input(type="text") Foo
```
But, if passed through `pug` will yield the following error:
```
input is a self closing element: <input/> but contains nested content.
```
I think that everything that would make `pug` fail to compile should be caught by `pug-lint`, when possible.

I'm using `pug-lint` v2.4.0 on Ubuntu (installed through `yarn`).
